openedx/features/enterprise_support/api.py

```diff
@@ -441,7 +444,9 @@ def get_enterprise_consent_url(request, course_id, user=None, return_to=None, en
     else:
         return_path = reverse(return_to, args=(course_id,))
 
+    enterprise_learner_details = get_enterprise_learner_data(request.site, user)
```

Comment:
Since we're just using the UUID, why aren't we using the `enterprise_customer_uuid_for_request` function created above?

Reply:
I created that function to reuse it here thinking it'd work, but it turns out getting it from the request doesn't work in all cases; it simply doesn't return the enterprise customer unless you're in an SSO scenario or something, which is I guess why it was originally made, based off of reading its code. But I think it's okay to leave it created, as it's probably correct to have it as a separate, reusable function for future use cases.

Reply:
Ok... so it looks like this was caused by https://github.com/edx/edx-platform/commit/4040c45f0646a4706dc56b5cfadb6c41e77ee16d#diff-d27e76023a81690ef221162664413e70 and the fix is now available at https://github.com/edx/edx-platform/pull/16163/files#r142703962 (see recent HipChat for more details). I guess I'll wait for that to merge, then we can rebase and use that fix.

Reply:
Looks like that was merged. Rebasing, etc..